Dockerfile

```
@@ -1,4 +1,4 @@
-FROM node:14-alpine
+FROM node:16
```

I would prefer to keep using the `alpine` image. More context about the [image variants](https://hub.docker.com/_/node#:~:text=to%2Ddate%20documentation.-,Image%20Variants,-The%20node%20images) for node on Dockerfile

```suggestion
FROM node:16-alpine
```